src/groundStation/commandParser.py

```diff
@@ -152,12 +152,10 @@ def __argCheck(self, args, inoutInfo, subservice=None):
 
     def __lexer(self, input):
         tokenList = []
-<<<<<<< HEAD:src/commandParser.py
+        # '([a-zA-Z_-]+|[\(\)]|[0-9_.-]*)' #([^#]*)#|
         tmp = re.split('([-.]+|[0-9.]+|[a-zA-Z0-9_-]+|[\(\)])', input)
-=======
-        tmp = re.split('([a-zA-Z_-]+|[\(\)]|[0-9_.-]*)', input)
->>>>>>> upstream/master:src/groundStation/commandParser.py
-        [tokenList.append(x.upper()) for x in tmp if not str(x).strip() == '']
+        [tokenList.append(x.upper()) for x in tmp if not (
+            str(x).strip() == '' or str(x).strip() == ',')]  # to accept ',' as delimiter
```

Comment:
My intention for this was to become able to use space in the strings and use a comma instead to separate the input arguments. It was an unsuccessful attempt but will be needed if we really consider changing the morse beacon or the beacon message from GS.

Reply:
Example: set_beacon_msg(I am Ex-Alta2) and put the "I am Ex-Alta2" into the packet as a single string.